curation-migration-datasync/helpers/hydrator.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  AddScheduledItemPayload,
+  CuratedFeedItem,
+  CuratedFeedProspectItem,
+  CuratedFeedQueuedItems,
+  ScheduledSurfaceToFeedIdMap,
+} from '../types';
+import {
+  convertDateToTimestamp,
+  getCuratorNameFromSso,
+} from '../eventConsumer';
+
+export function hydrateCuratedFeedItem(
+  queuedItem: CuratedFeedQueuedItems,
+  scheduledDate: string
+): CuratedFeedItem {
+  return {
+    queued_id: queuedItem.queued_id,
+    feed_id: queuedItem.feed_id,
+    status: 'live',
+    prospect_id: queuedItem.prospect_id,
+    resolved_id: queuedItem.resolved_id,
+    time_added: queuedItem.time_added,
+    time_updated: queuedItem.time_updated,
+    time_live: convertDateToTimestamp(scheduledDate),
+    //will be populated from the database
+    curated_rec_id: 0,
```

Comment:
I wouldn't include this on the object -- and all the other ones below that are the same

Reply:
i first kept this as optional, but when i assign it to other object where they are mandatory field, then it throws an error `cannot assingn undefined to number`.

for e.g
i can set prospect_id in `CuratedFeedProspectItem` as optional field, but when i use them, i need to set it like this:  `CuratedFeedQueuedItems.prospect_id = CuratedFeedProspectItem.prospect_id ?? null`

Reply:
ah made them optional, and checking for undefined  before assigning removes the error

Reply:
If the input doesn't have that field, but the output does, you can just use `Omit<...>`

Reply:
we then need to maintain two types for the same entity, if optional doesn't have big side effect, can we retain the current implementation?